oneargopy/Settings.py

```diff
@@ -31,7 +31,7 @@ class DownloadSettings():
         :param: user_settings : str - An optional path to user defined 
             settings.
 
-        Settings: 
+        SETTINGS: 
```

Comment:
WHY DO YOU USE ALL CAPS HERE???

Reply:
I am, not sure why I did that?? I'll switch it back to lowercase so we're not yelling at anyone haha.